packages/utils/rbac.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import {
+  AccessReviewResourceAttributes,
+  k8sCreate,
+  K8sVerb,
+} from '@openshift-console/dynamic-plugin-sdk';
+import * as _ from 'lodash-es';
+import log from 'loglevel';
+import { useSafetyFirst } from '../components/safety-first';
+import { ProjectModel, SelfSubjectAccessReviewModel } from '../models';
+import { SelfSubjectAccessReviewKind } from '../types';
+
+// Memoize the result so we only make the request once for each access review.
+// This does mean that the user will have to refresh the page to see updates.
+// Accept an `impersonateKey` parameter to include in the cache key even though
+// it's not used in the function body. (Impersonate headers are added
+// automatically by `k8sCreate`.) This function takes in the destructured
+// resource attributes so that the cache keys are stable. (`JSON.stringify` is
+// not guaranteed to give the same result for equivalent objects.)
```

Comment:
nits: we usually use /** */ so that it will be helpful if someone hovers over the function name.

Reply:
Done.